### PR TITLE
Migrate GetPeersScore to MigratingEndpointAdapter

### LINKED ISSUE #5799: Migrate GetPeersScore to MigratingEndpointAdapter

### PR DESCRIPTION
**Pull request by courtneyeh**
fixes #5799 

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/CoreTypes.java [hunk 1]
```diff
@@ -89,6 +89,16 @@
           .format("integer")
           .build();
 
+  public static final StringValueTypeDefinition<Double>
+      DOUBLE_TYPE = // TODO make raw type not string
+      DeserializableTypeDefinition.string(Double.class)
+              .formatter(Object::toString)
+              .parser(Double::valueOf)
```
[the remark this answers is not in the document]
Comment (rolfyone) [reply infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/CoreTypes.java]: if its easier here we can leave it as string, I'm not too worried either way...

[the remark this answers is not in the document]
Comment (courtneyeh) [reply infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/CoreTypes.java]: Okay perfect, I'll leave it

File: infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/DoubleTypeDefinition.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import com.fasterxml.jackson.core.JsonParser;
+import java.io.IOException;
+import java.util.Objects;
+
+class DoubleTypeDefinition extends PrimitiveTypeDefinition<Double> {
+
+  @Override
+  public void serializeOpenApiTypeFields(final JsonGenerator gen) throws IOException {
+    gen.writeStringField("type", "number");
+  }
```
Comment (rolfyone) [inline infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/DoubleTypeDefinition.java]: check in the string integer type how we write the subtype (format) - we can set format to `double` looking at the spec..
https://swagger.io/specification/

Comment (courtneyeh) [reply infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/DoubleTypeDefinition.java]: Added a format field.

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**